wd_helper.au3

```diff
@@ -902,11 +908,12 @@ EndFunc   ;==>_WD_FrameList
 ; #INTERNAL_USE_ONLY# ===========================================================================================================
 ; Name ..........: __WD_FrameList_Internal
 ; Description ...: function that is used internally in _WD_FrameList, even recursively when nested frames are available
-; Syntax ........: __WD_FrameList_Internal($sSession, $sLevel, $sFrameAttributes, $bIsHidden)
+; Syntax ........: __WD_FrameList_Internal($sSession, $sLevel, $sFrameAttributes, $bIsHidden[, $iTimeout = Default])
```

Comment:
It's not a big deal, but there's really no reason for $iTimeout to be optional since it is always called with a value.

Reply:
Please take a look: [__WD_FrameList_Internal - $iTimeout](https://github.com/Danp2/au3WebDriver/pull/456/commits/e6dc5786d26cb4410a47fd5af9acaab934b344f5)